docs/labs/flockercontainers.rst

```diff
@@ -0,0 +1,96 @@
+.. _flocker-containers:
+
+=============================
+Running Flocker in Containers
+=============================
+
+The :ref:`Flocker Installer <labs-installer>` allows you install Flocker in a variety of environments.
```

Comment:
I'd probably remove this part. The Labs installer only supports a single environment, Ubuntu-14.04 + AWS:
 * https://docs.clusterhq.com/en/1.8.0/labs/installer.html#supported-configurations